src/Compilers/Core/Portable/PEWriter/MetadataWriter.cs

```diff
@@ -2808,6 +2815,35 @@ private void PopulateStandaloneSignatures()
             }
         }
 
+        private int[] SerializeThrowNullMethodBodies(BlobBuilder ilBuilder, out Blob mvidStringFixup)
```

Comment:
It is only used to ensure that the caller's `mvidStringFixup` is initialized on this code path.